sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -20,6 +20,7 @@ dayjs.extend(isSameOrAfter);
 
 export const DATE_ONLY_ISO_FORMAT = "YYYY-MM-DD";
 export const DATE_ONLY_FORMAT = "MMM DD YYYY";
+export const DATE_ONLY_FULL_MONTH_FORMAT = "MMMM D, YYYY";
```

Comment:
I think this should be "MMMM DD, YYYY" to take account of the two digit dates.

Reply:
"MMMM DD, YYYY" will format as "April 07, 2020":
![image](https://github.com/user-attachments/assets/226aaea1-388d-4779-8fbc-6457755ae831) 
while "MMMM D, YYYY" will format as "April 7, 2020":
![image](https://github.com/user-attachments/assets/20ac20e3-8fec-4500-8391-4e4b85efeb06)

Reply:
![image](https://github.com/user-attachments/assets/7aed1c02-614f-4c41-8c50-4a57319524ea)